docs/GettingStarted.md

```diff
@@ -355,6 +355,16 @@ services (S3, ElastiCache etc.).
 
     Aws::S3::Client.new.list_buckets # traced call
 
+### Dalli
+
+Dalli integration will trace all calls to your `memcached` server:
+
+    require 'dalli'
+    require 'ddtrace'
```

Comment:
don't we need to patch / patch all something?

Reply:
will add that as a general example; unfortunately for Rails it will be:
```ruby
require 'ddtrace'

# Tracing configuration
Rails.configuration.datadog_trace = {
  auto_instrument: true,
  auto_instrument_redis: true,
  auto_instrument_grape: true,
  default_service: ENV['RAILS_SERVICE'] || 'rails-local',
  default_database_service: ENV['RAILS_MYSQL_SERVICE'] || 'db-local',
  default_cache_service: ENV['RAILS_CACHE_SERVICE'] || 'cache-local',
  default_controller_service: ENV['RAILS_CONTROLLER_SERVICE'] || 'controller-local',
  trace_agent_hostname: ENV['DATADOG_TRACER'] || 'datadog'
}

Datadog::Monkey.patch_module(:dalli)
pin = Datadog::Pin.get_from(::Dalli)
pin.service = 'dalli-cache'
```

mostly because we have some things that are patched via Rails settings, and others that are not. I'd say to not add Dalli to the `auto_instrument_*` since we're planning to remove this settings in favor of a better way to configure our integrations.